extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,40 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
+  html_format: AMP
+  tag_name: "IMG"
+  spec_name: "AMP-STORY-PLAYER IMG"
+  mandatory_ancestor: "AMP-STORY-PLAYER"
```

Comment:
Consider mandatory_ancestor_suggested_alternative until wider support for IMG arrives.